src/plays/countries-statics/Readme.md

```diff
@@ -0,0 +1,27 @@
+# countries-statics
+
+Fetch country specific statistic such as demographics by single click on map.
```

Comment:
This is a great play and an advanced one. Please provide some more technical information about it here.

Reply:
I don't want the description to be long so I provided additional details in ## Implementation Details